docs/extensions/consensus/04-extension.mdx

```diff
@@ -0,0 +1,57 @@
+---
+sidebar_position: 4
+sidebar_label: "Extension-based Changes"
+id: fork-extensions
+title: "Extension-based Changes"
+description:  "Extension-based system for coordinating activation of consensus rule changes with new software."
+slug: /extensions/fork-ext
+---
+
+:::caution
+The coordinated upgrade system and hardfork definitions defined here are
+**experimental**. The system is likely to change in the future as use cases are
+evaluated.
+:::
+
+### Defining the Changes
+
+A set of well-defined changes may be specified using `kwild`'s extension system,
+which allows network operators to customize a network without forking or
+modifying the `kwild` source. The `extensions/consensus.Hardfork` Go structure
+is used to define several well-defined types of changes that are exposed by its
+fields. See the
+[godocs](https://pkg.go.dev/github.com/kwilteam/kwil-db/extensions/consensus)
```

Comment:
Link does not seem to work, however I believe it should after we merge and tag?

Reply:
Yeah, I think it'll be right after there's a tag where this exists.

Reply:
Looks like godoc found it: https://pkg.go.dev/github.com/kwilteam/kwil-db/extensions/consensus